composer.json

```diff
@@ -20,6 +20,7 @@
     "require": {
         "php": "^7.1",
         "illuminate/support": "^5.5 | ^6.0",
+        "symfony/process": "^4.3.4",
```

Comment:
This need to be "^4.3 | ^5.0"

Reply:
I thought you wanted to do that on the release where you also update illuminate/support to | ^7.0

Going to add it now